bin/node/src/chain_spec.rs

```diff
@@ -334,7 +335,7 @@ fn generate_genesis_config(
             validator_count: authorities.len() as u32,
             // to satisfy some e2e tests as this cannot be changed during runtime
             minimum_validator_count: 4,
-            invulnerables: to_account_ids(&authorities).collect(),
+            invulnerables: to_account_ids(&authorities[..2]).collect(),
```

Comment:
This seems like it could _theoretically_ fail at runtime

Reply:
this is called at the point of bootsraping the chain so this is not executed in the runtime. Also if we have less than 2 authorities it doesn't make sense to run the chain where the minimum validator count is equal to 4 among other stuff

Reply:
Fair enough 🙂